data/json/itemgroups/SUS/domestic.json

```diff
@@ -667,5 +667,105 @@
       { "collection": [ { "item": "candle", "count": [ 1, 2 ] }, { "item": "matches" } ], "prob": 20 },
       { "item": "toilet_paper", "prob": 80 }
     ]
+  },
+  {
+    "id": "SUS_desks_bedroom_unisex",
+    "type": "item_group",
+    "//": "clothing and wearable items for domestic desks, tables, etc.",
+    "subtype": "collection",
+    "items": [
```

Comment:
`entries` should be used instead of `items` here. `items` is supported for legacy itemgroups.